cmd/main.go

```diff
@@ -31,8 +31,7 @@ import (
 var globalFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "config-dir, C",
-		Value: getConfigDir(),
-		Usage: "Path to configuration directory.",
+		Usage: "User specified path to configuration directory.",
```

Comment:
If home is probable, you have to set default config directory.  Now it takes current directory as default directory for command `./minio server /tmp/d1`